### PR TITLE
chore: validate() tests for invalid NIL uuids

### PR DESCRIPTION
**Pull request by broofa**
Triggered by #581

cc: @bitpup

### CODE REVIEW

**Comment by bitpup** [comment]
asleep at the wheel sorry

File: test/unit/validate.test.js [hunk 1]
```diff
@@ -20,7 +20,18 @@ describe('validate', () => {
 
     assert.strictEqual(validate('invalid uuid string'), false);
 
-    assert.strictEqual(validate('00000000000000000000000000000000'), false);
```
Comment (LinusU) [inline test/unit/validate.test.js:23]: Maybe keep this line as well?

**Review by LinusU — APPROVED** [review]
Nice 👍

File: test/unit/validate.test.js (hunk 1 above)
Comment (broofa) [reply test/unit/validate.test.js:23]: "Oops" (fixed)

**Review by ctavan — APPROVED** [review]
LGTM

**Pull request merged**